server/controllers/questionController.js

```diff
@@ -1,12 +1,16 @@
-const mongoose = require('mongoose');
 const Question = require('../models/Question');
-const User = require('../models/User');
 
 module.exports = {
   create: async (req, res) => {
-    const { title, difficulty } = req.body;
+    const { index, url, title, difficulty, description } = req.body;
```

Comment:
createQuestion would be a better name - we always want to be specific as possible. But are we actually using this anywhere?